style.css

```
@@ -15,6 +15,37 @@ a {
     text-decoration: none;
 }
 
+.main-header {
+    background-color: #eeeeee;
+    padding-bottom: 30px;
+}
+
+.main-navigation {
+    padding-top: 45px;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.container {
+    width: 1160px;
+    margin-left: auto;
+    margin-right: auto;
+    display: flex;
+}
+
+.navigation ul {
+    margin: 0;
+    padding: 0;
+}
```

```suggestion
.navigation {
    margin: 0;
    padding: 0;
}
```
`.navigation` это и есть сам `ul`